contentcuration/contentcuration/static/js/edit_channel/models.js

```diff
@@ -215,20 +214,31 @@ var ContentNodeCollection = BaseCollection.extend({
     	});
     	return promise;
     },
-    move:function(target_parent, sort_order){
+    move:function(target_parent){
     	var self = this;
-		var promise = new Promise(function(resolve, reject){
-			self.forEach(function(model){
-				model.set({
-					parent: target_parent.id,
-					sort_order:++sort_order
-				});
-	    	});
-	    	self.save().then(function(collection){
-	    		resolve(collection);
-	    	});
-		});
-        return promise;
+    	var promise = new Promise(function(resolve, reject){
+    		var node_list = [];
+    		self.forEach(function(node){
+    			node_list.push('{"id":"' + node.get('id') + '","sort_order":' + node.get('sort_order') + '}');
```

Comment:
For sort order, it's mostly in case a user drops nodes in between nodes. Right now, the client side will calculate sort orders that are between the nodes, but the server side logic does a get, which overrides the newly calculated order